app/views/customers/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= @customer.name %>
+
+<% @customer.items.each do |item| %>
+  <h3><%= item.name %></h3>
+  <%= item.price %>
+  <%= item.get_supermarket_name %>
+<% end %>
+
+<%= form_with url: "/customers/#{@customer.id}", method: :patch, data: { turbo: false } do |f| %>
```

Comment:
So to make this route more RESTful and to utilize our join association we will want to make this route
```ruby 
"/customers/#{customer.id}/items", method: :patch
```
Which you will then want to point to a `customer_items#create` in your routes.